torch/_higher_order_ops/map.py

```diff
@@ -253,6 +243,17 @@ def expand_tensor(t):
             next_name = candidate
 
     proxy_mode.tracer.root.register_module(next_name, body_graph)
+
```

Comment:
Because now the tracer is based on module path, we do need to register the node before actually executing it